docker-compose.yml

```diff
@@ -6,7 +6,6 @@ services:
     environment: 
       OPENAI_API_KEY: ${OPENAI_API_KEY}
       TAVILY_API_KEY: ${TAVILY_API_KEY}
-      DOC_PATH: "=./my-docs"
```

Comment:
Shouldn't it be DOC_PATH: ${DOC_PATH} ?

Reply:
No need. Relying on .env is preferable

Path variables get tricky and run into edge cases when dealing with containers.

Better to play it safe and work with what we know works until.peoven otherwise